c-pallets/file-bank/src/lib.rs

```diff
@@ -472,6 +480,8 @@ pub mod pallet {
 			})?;
 
 			//To be tested
+			//because of this line, your weight benchmark should depend on
+			//the lenght of the slice_info argument
```

Comment:
Yes, you are right. I should calculate weight according to slice.
I modified the benchmark code here:
https://github.com/CESSProject/cess/pull/88/files#diff-127d5bc8d5ced709e3871ca77106ae0b18c87710ea3eae9e16437a826d0d603bR267-R296